revolv/project/views.py

```diff
@@ -56,6 +59,11 @@ def form_valid(self, form):
             return self.render_to_response(self.get_context_data(form=form))
 
         return super(CreateProjectView, self).form_valid(form)
+    
+    def form_invalid(self, form):
```

Comment:
Oh oops, we really don't need this function since we're not doing the dynamic donation levels, right?